core/store/src/lib.rs

```diff
@@ -60,7 +60,9 @@ pub const COL_LAST_APPROVALS_PER_ACCOUNT: Option<u32> = Some(21);
 pub const COL_MY_LAST_APPROVALS_PER_CHAIN: Option<u32> = Some(22);
 pub const COL_STATE_PARTS: Option<u32> = Some(23);
 pub const COL_EPOCH_START: Option<u32> = Some(24);
-const NUM_COLS: u32 = 25;
+// Map account_id to announce_account
```

Comment:
Now that our routing table is persisted in the storage, what will happen if I shutdown a node and come back a year later and restart a node which make it pick up a super outdated routing table? Will it be able to recover even though the routing table would be completely invalid?

Reply:
Totally, data is going to be get from persisted storage to cache on demand, so old data will not get into cache unless it is required. Moreover, there is no such thing as old data becoming invalid, it just got outdated but it can't cause any harm other than consuming space.